<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>order</class>
 <widget class="QDialog" name="order">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1507</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_title_Form">
     <property name="maximumSize">
      <size>
       <width>291</width>
       <height>84</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="ImgReport_2">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Recource.qrc">:/rec/img/Form.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbReport_2">
        <property name="minimumSize">
         <size>
          <width>201</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>201</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Форма Заказы</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>571</width>
       <height>411</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>571</width>
       <height>411</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lbID_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>61</y>
        <width>78</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Номер заказа </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>64</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Дата Заказа</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbName_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>219</y>
        <width>78</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> ФИО Клиента</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbINN_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>298</y>
        <width>95</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ФИО Сотрудника</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>377</y>
        <width>40</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Оплата</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cBpay">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>377</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>22</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>true</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>false</string>
       </property>
      </item>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>441</width>
        <height>371</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>441</width>
        <height>371</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>441</width>
        <height>371</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="leOrder_number"/>
       </item>
       <item>
        <widget class="QLineEdit" name="leDate_2"/>
       </item>
       <item>
        <widget class="QComboBox" name="cbFio_client"/>
       </item>
       <item>
        <widget class="QComboBox" name="cbFio_emp"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame_3">
     <property name="maximumSize">
      <size>
       <width>491</width>
       <height>44</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnAdd_2">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEdit_2">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDel_2">
        <property name="text">
         <string>Del</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemove_2">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnReport_order">
        <property name="text">
         <string>Отчет о заказах</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QFrame" name="frame_4">
     <property name="minimumSize">
      <size>
       <width>351</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>351</width>
       <height>41</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btnBack">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Назад </string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QTextEdit" name="teResult">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>121</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="4">
    <widget class="QTableWidget" name="twOrg">
     <property name="minimumSize">
      <size>
       <width>871</width>
       <height>561</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Recource.qrc"/>
 </resources>
 <connections/>
</ui>
